update-lb4-docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+ORG=strongloop
+REPO=loopback-next
+DOCS=./pages/en/lb4
+BRANCH=master
+ARCHIVE_NAME="loopback-next-$BRANCH"
+
+mkdir -p $DOCS
+# pull down all the markdown files (reserving dir structure)
+# given the GitHub ORG, REPO, and BRANCH
+#curl -L https://github.com/$ORG/$REPO/archive/$BRANCH.tar.gz | tar --strip-components 1 -C "$DOCS" --wildcards \
+# -xzvf- "$ARCHIVE_NAME/docs/*.md" "$ARCHIVE_NAME/packages/*.md"
+# "$ARCHIVE_NAME/docs/*.png"
+npm i @loopback/docs --prefix $DOCS
```

Comment:
I think it can be further simplified if we add `@loopback/docs` as a dependency in `package.json` of `loopback.io`. This way, `npm install` at the root level will pull in `node_modules/@loopback/docs` and we can add a post-install script to copy them to the right place.
_

Reply:
I talked to @rmg and if we aim to do that, then we should probably introduce Travis to this repo since the Jenkins pipeline is
>running directly on the Jenkins master and that server doesn't have any external executors 

and doesn't have `node/npm` installed. It would be a hassle to install them there. Also,
>all GitHub Pages does is run Jekyll and upload the generated `_site` directory (or whatever it is..) to S3